tests/jballerina-unit-test/src/test/resources/test-src/functions/functions_with_included_record_parameters_negative.bal

```diff
@@ -279,3 +279,21 @@ function testInvokingFunctionContainingIncludedRecordParamOfTypeOtherThanRecordO
     _ = obj.fn(f = {});
     _ = obj.fn2(f = {});
 }
+
+type Record5 record {};
+
+function fn5(*Record5 record5) {
+}
+
+function testInvokingFunctionWithNamedArgForIncludedRecordParam() {
+    fn5(record5 = {}, j = 2); // error
+    fn5(j = 2, record5 = {}); // error
+}
+
```

Comment:
Shall we also add a test for when there are more than one open records as included record parameters?

Reply:
Updated.